executor/api/grpc/seldon/server.go

```diff
@@ -56,9 +56,60 @@ func (g GrpcSeldonServer) SendFeedback(ctx context.Context, req *proto.Feedback)
 	return payloadToMessage(resPayload), nil
 }
 
+func (g GrpcSeldonServer) Metadata(ctx context.Context, req *proto.SeldonModelMetadataRequest) (*proto.SeldonModelMetadata, error) {
```

Comment:
Seems like it would be worth keeping consistency of "ModelMetadata" and "GraphMetadata" across the codebase

Reply:
I think the method here must corresponds to the definition of the service in the proto. Would have to adjust there.

Reply:
Added docstring to clarify - if you think it is better to rename now is the time as probably it's better to avoid modifying protos after this is merged.

Reply:
Renamed corresponding `rpc` to `ModelMetadata`.